controllers/consoleplugin/config/static-frontend-config.yaml

```
@@ -904,6 +904,121 @@ filters:
     name: Network events flow monitoring
     component: text
     hint: Specify a single network event.
+scopes:
+  - id: cluster
+    name: Cluster
+    shortName: Cl
+    description: Cluster name or identifier
+    labels:
+      - K8S_ClusterName
+    feature: multiCluster
+    filter: cluster_name
+  - id: zone
+    name: Zone
+    shortName: AZ
+    description: Availability zone
+    labels:
+      - SrcK8S_Zone
+      - DstK8S_Zone
+    feature: zones
+    groups:
+      - clusters
+    filters:
+      - src_zone
+      - dst_zone
+  - id: host
+    name: Node
```

shortName is missing for Node

```suggestion
    name: Node
    shortName: Node
```

---

you removed it in the last commit :-)